app/__tests__/app.test.js

```diff
@@ -0,0 +1,51 @@
+const request = require("supertest")
+const app = require("../app")
+const db = require("../../db/connection")
+const data = require("../../db/data/test-data/index")
+const seed = require("../../db/seeds/seed")
+
+beforeEach(() => {
+    return seed(data);
+})
+afterAll(() => {
+    return db.end();
+})
+
+describe("GET /topics", () => {
+    describe("-- functionality tests", () => {
+        test("200: responds with a status of 200 when there is an appropriate request", () => {
+            return request(app)
+            .get("/api/topics")
+            .expect(200)
+        })
+        test("200: responds with an array of topic objects", () => {
+            return request(app)
+            .get("/api/topics")
+            .expect(200)
+            .then(({body}) => {
+                expect(Array.isArray(body.topics)).toBe(true)
```

Comment:
We should not test for this if we later use anything that would imply that we are testing an array 🙂